README.md

```diff
@@ -2,8 +2,50 @@
 
 [![Build Status](https://travis-ci.org/measurement-kit/measurement-kit.svg?branch=master)](https://travis-ci.org/measurement-kit/measurement-kit)
 
-MeasurementKit is an experimental library that provides common functionalities
-useful to implement open measurement tools on mobile platforms.
+MeasurementKit is a library that implements open measurement methodologies
```

Comment:
I would specify we are talking about network measurements.

Perhaps it's also good to clarify that users of the library may be either those that want the results of the measurements to be used in the application logic of some arbitrary piece of software or it may be of interested to researchers or people that have specific network measurement needs.